identify/extensions.py

```diff
@@ -151,6 +151,7 @@
     'md': {'text', 'markdown'},
     'mdx': {'text', 'mdx'},
     'meson': {'text', 'meson'},
+    'metal': {'text', 'metal-shading-language'},
```

Comment:
it's more common now ot have the identify tags match the extension itself rather than something else